asn1/CommonLedgerProtocol.asn

```diff
@@ -21,81 +21,53 @@ SideProtocolData ::= SEQUENCE OF SEQUENCE {
   protocolData OCTET STRING
 }
 
-PacketRequest ::= SEQUENCE {
-  packet InterledgerPacket,
+Ack ::= SEQUENCE {
+  ack NULL,
```

Comment:
With the messages defined here I don't see any ambiguity about the responses.

When you send any of `Prepare`, `Fulfill`, `Reject`, `Message`, you either get an `Ack` or a `Response` back. If you set the `requestId` in the `CommonLedgerProtocolPacket` to `0`, you are indicating you don't want that response. When you send a `CustomRequest` you'll probably get a `CustomResponse`, unless otherwise defined in the side protocol.